frontend/frontend.go

```diff
@@ -0,0 +1,23 @@
+package frontend
+
+import (
+	"github.com/labstack/echo/v4"
+	"github.com/labstack/echo/v4/middleware"
+)
+
+func RegisterHandlers(e *echo.Echo) {
```

Comment:
This doesn't seem to be needed with Wails?

Reply:
Update: Wails only makes sense for Breez and we can have different ways to launch the app.